contracts/interfaces/IGnosisSafe.sol

```diff
@@ -0,0 +1,45 @@
+//SPDX-License-Identifier: Unlicense
+pragma solidity ^0.8.0;
+
+import "@gnosis.pm/safe-contracts/contracts/common/Enum.sol";
+
+interface IGnosisSafe {
```

Comment:
@tbwebb22 does this interface not exist in the `@gnosis.pm/safe-contracts` import?

Reply:
Nope it doesn't, not sure why.